x-pack/plugins/security_solution/public/resolver/view/resolver_without_providers.tsx

```diff
@@ -58,7 +59,7 @@ export const ResolverWithoutProviders = React.memo(
       shouldUpdate,
       filters,
     });
-
+    useAutotuneTimerange();
```

Comment:
Do you think there need to be tests modified/added to accomodate this new hook.

Reply:
Resolver react tests are currently not working due to a bug in simulator, will add some tests for this once that has been resolved.